class.jetpack-cli.php

```diff
@@ -148,9 +150,9 @@ public function disconnect( $args, $assoc_args ) {
 			case 'user':
 				if ( Jetpack::unlink_user( $user->ID ) ) {
 					Jetpack::log( 'unlink', $user->ID );
-					WP_CLI::success( sprintf( __( '%s has been successfully disconnected.', 'jetpack' ), $action ) );
+					WP_CLI::success( sprintf( __( 'User has been successfully disconnected.', 'jetpack' ), $action ) );
 				} else {
-					WP_CLI::error( sprintf( __( '%s could not be disconnected.  Are you sure they\'re connected currently?', 'jetpack' ), "{$user->login} <{$user->email}>" ) );
+					WP_CLI::error( sprintf( __( 'User could not be disconnected.  Are you sure they\'re connected currently?', 'jetpack' ), "{$user->login} <{$user->email}>" ) );
```

Comment:
Yes, what I meant was that in the proposed change we're using `sprintf`, but passing the string without placeholders.